src/components/Notifications/SilenceNotificationForm/NotificationSilenceForm.tsx

```diff
@@ -37,6 +38,39 @@ export default function NotificationSilenceForm({
   onSuccess = () => {},
   onCancel = () => {}
 }: NotificationSilenceFormProps) {
+  const [showFilterExamples, setShowFilterExamples] = useState(false);
+  const [showSelectorsExamples, setShowSelectorsExamples] = useState(false);
+
+  const filterExamples = [
+    {
+      code: `config.type == "AWS::RDS::DBInstance" && config.tags["account-name"] == "flanksource" && config.config.Engine == "postgres"`,
```

Comment:
@adityathebe - Can we add more examples around silencing health checks based on a tag e.g. Expected-Fail: true,     resources of a helm chart,  pod starting with a name